docker/release/Dockerfile

```diff
@@ -0,0 +1,67 @@
+#
+# Postfacto, a free, open-source and self-hosted retro tool aimed at helping
+# remote teams.
+#
+# Copyright (C) 2016 - Present Pivotal Software, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+#
+# it under the terms of the GNU Affero General Public License as
+#
+# published by the Free Software Foundation, either version 3 of the
+#
+# License, or (at your option) any later version.
+#
+#
+#
+# This program is distributed in the hope that it will be useful,
+#
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+#
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#
+# GNU Affero General Public License for more details.
+#
+#
+#
+# You should have received a copy of the GNU Affero General Public License
+#
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+FROM node:12.6.0 as front-end
+
+COPY ./web /web
+WORKDIR /web
+
+RUN npm install
```

Comment:
It's quicker to `npm ci` here. Also if you copy over the `package{,-lock}.json` separately first then install, _then_ copy the rest of the files, it reduces the subsequent build times for any changes that _don't_ require dependency changes.

Reply:
Switched to `npm ci` but not sure I follow how copying the package file helps, lets do that as an optimization after the fact